example-jobs/ghe_job.groovy

```diff
@@ -0,0 +1,32 @@
+import jenkins.automation.builders.BaseJobBuilder
+import jenkins.automation.utils.GhUtils
+
+new BaseJobBuilder(
+    name: 'GH_PR_builder',
+    description: 'Does GH PR building',
+).build(this).with {
+    GhUtils.ghPrWatcher(
+        delegate,
+        [
+            ghProject: 'cfpb/reponame',
+            ghHostname: 'github.org.tld',
+            ghAuthId: '<unique GH PR builder credential ID>',
+            ghPrHooks: false,
+            ghPrCron: '*/2 * * * *',
+            ghPrOrgsList: 'CFPB',
+            ghPrStatusContext: 'Test your example job',
+            ghPrResultMessage: [
+              'SUCCESS': 'Tests completed normally',
+              'ERROR': 'Tests errored',
+              'FAILURE': 'Tests failed',
+            ]
+        ]
+    )
+
+    steps {
+        shell('''
+        sh test.sh
```

Comment:
I think this will fail our travis build, why not just use 'echo'?

Reply:
I don't see the Travis build running anything other than:

```
  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
```
And the build appears to have passed.

It's not even performing any gradle steps in this repository. Should it be running `build` or something? I definitely don't understand the idea of rendering and also running Jenkins jobs via Travis in this repo's current state.